play23/src/main/scala/play/api/test/ops/AsyncResultExtractors.scala

```diff
@@ -0,0 +1,123 @@
+package play.api.test.ops
+
+import play.api.http.HeaderNames._
+import play.api.http.Status._
+import play.api.libs.iteratee.Iteratee
+import play.api.libs.json.{JsValue, Json}
+import play.api.mvc._
+import play.twirl.api.Content
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * Similar to play.api.test.ResultExtractors except that it is designed to work better
```

Comment:
I removed the link because there wasn't a compile-time dependency on that library. Only in the test code.